.github/workflows/smoke-test-node-api.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-node@v3
         with:
-          node-version: latest
+          node-version: 'lts/*'
```

Comment:
I was getting [some errors](https://github.com/chromaui/chromatic-cli/actions/runs/11373960183/job/31642317236) in this action related to

```
YN0018: resolve@patch:resolve@npm%3A1.22.8#optional!builtin<compat/resolve>::version=1.22.8&hash=c3c19d: The remote archive doesn't match the expected checksum
```

Turns out, it was because the `latest` version of Node was bumped to v23.0.0. We can set this to `lts` to fix that issue. I don't think we need to test against `latest` but I'm open to changing this back.